src/v/config/configuration.cc

```diff
@@ -1545,7 +1546,35 @@ configuration::configuration()
       "Maximum capacity of rate limit accumulation"
       "in controller configuration operations limit",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
-      std::nullopt) {}
+      std::nullopt)
+  , kafka_throughput_limit_node_in_bps(
```

Comment:
fwiw I think we can merge this even before we fully resolve the conversation that is happening in parallel related to cluster vs node level config options for 23.1.

Reply:
sure